example/spec/parallel/testAPI/granuleSpec.js

```diff
@@ -101,6 +102,14 @@ describe('The Granules API', () => {
         execution: undefined,
         files: [granuleFile],
       }));
+      randomBareGranuleRecord = removeNilProperties(fakeGranuleFactoryV2({
```

Comment:
Nit: takes time to understand what 'bare' means out of context, random is probably not useful when referring to a factory produced test object. 

Consider like granuleRecordWithoutFiles.

Reply:
Also: why the overhead of a separate granule, explicitly without files in the units here?

Reply:
this is a good point. I cleaned it up
https://github.com/nasa/cumulus/pull/3853/commits/aa555d36d9f4d21476727fae9662643d2b7db767